### ISSUE TITLE
[Query API] Location Query

### ISSUE THREAD

**Issue by LukasPietzschmann** [flowr-analysis/flowr#1041]
Query for the location of a specific Node (given by its ID)

**Comment by LukasPietzschmann** [comment]
@EagleoutIce how do you imagine the grouping to work? Should it be similar to `call-context-query` with _kind_ and _subkind_?

**Comment by EagleoutIce** [comment]
No, just return one result which contains a list of all requested vertices, essientially similar to the id-map but filered for the ids of interest.

**Comment by LukasPietzschmann** [comment]
Alright. But I think it would be better to return a map (id -> location) instead of a plain list of locations. This way, the client does not need to remember the order of the individual location-queries.

**Comment by EagleoutIce** [comment]
this was what i meant with the second part, sorry :D

**Comment by EagleoutIce** [comment]
The map is now produced by the [location map query](https://github.com/flowr-analysis/flowr/wiki/Query-API#location-map-query)
